sdk/tables/Azure.Data.Tables/src/TableClient.cs

```diff
@@ -554,11 +554,11 @@ public virtual Response UpdateEntity(IDictionary<string, object> entity, string
         /// Queries entities in the table.
         /// </summary>
         /// <param name="filter">Returns only tables or entities that satisfy the specified filter.</param>
-        /// <param name="top">Returns only the top n tables or entities from the set.</param>
+        /// <param name="maxPerPage">Returns this number of tables or entities per page.</param>
```

Comment:
suggestion: `The maximum number of items that will be returned per page.`

Reply:
Since this query is only for entities, would it better to write "entities" instead of "items"? And then edit the other descriptions that use "tables or entities"?

Reply:
Sure, I think that makes sense to specify `entities` or `tables` where appropriate.